.github/workflows/automerge.yaml

```diff
@@ -0,0 +1,30 @@
+name: automerge
+on:
+  pull_request:
```

Comment:
As I understand this action will apply for all branches. Should we limit it for `stage` and `production` branches?